src/main/kotlin/com/motycka/edu/lesson04/TemperatureConverter.kt

```diff
@@ -1,44 +1,10 @@
 package com.motycka.edu.lesson04
 
-import io.github.oshai.kotlinlogging.KotlinLogging
-
-private val logger = KotlinLogging.logger { }
-
-/**
- * Utility object for converting temperature values between Fahrenheit and Celsius
- */
 object TemperatureConverter {
 
-    /**
-     * Converts temperature value given in Fahrenheit to Celsius
-     *
-     * @param fahrenheit temperature value in Fahrenheit
-     * @return temperature value in Celsius
-     * @see <a href="https://en.wikipedia.org/wiki/Fahrenheit">Fahrenheit</a>
-     * @see <a href="https://en.wikipedia.org/wiki/Celsius">Celsius</a>
-     */
-    fun toCelsius(fahrenheit: Double): Double {
-        logger.info { "Converting $fahrenheit Fahrenheit to Celsius" }
-        return (fahrenheit - 32) * 5 / 9
-    }
```

Comment:
You were not supposed to change the tested class. The assignment was to write tests for the class.